documentation/TESTING_PLAN.md

```diff
@@ -16,7 +16,16 @@ In general, we're approaching testing bottom-up, with each unit function tested
 
 ### Unit-level tests
 
-- TBD 10
+- Test that impassible walls are impassible by the player.
+- Test that the level is complete/congratulatory message when and only when all boxes are moved to designated goal cells.
```

Comment:
Congratulatory message is purely a JS/UI concept, so the game engine only needs to return true or false when the level is complete via the `solved()` func. Feel free to move this to the front-end integration tests though.